src/routes/admin/ieee8021x/constants.ts

```diff
@@ -0,0 +1,79 @@
+/*********************************************************************
+ * Copyright (c) Intel Corporation 2022
+ * SPDX-License-Identifier: Apache-2.0
+ **********************************************************************/
+
+export interface FormOption<T> {
+  value: T
+  label: string
+}
+
+// AMT Authentication ProtocoL
+// https://software.intel.com/sites/manageability/AMT_Implementation_and_Reference_Guide/HTMLDocuments/WS-Management_Class_Reference/AMT_8021XProfile.htm
```

Comment:
toolkit docs has a reference to AMT SDK documentation link, Do we still need it here? Also, we are not using AMT interface of IEEE8021xProfile. Please remove it.